### PR TITLE
Feature/add attributes

### LINKED ISSUE #81: Add attribute: held_by
|  |  |
| ---------- | ------------- |
| label           | **Held By** |
| name          |  **held by** |
| predicate   | **::RDF::Vocab::MADS.HierarchicalGeographic**

### LINKED ISSUE #82: Add attribute: alternative ("Alternative Title")
|  |  |
| ---------- | ------------- |
| label           | **Alternative Title** |
| name          |  **alternative** |
| predicate   | **::RDF::Vocab::DC.alternative**

### LINKED ISSUE #83: Add attribute: abstract
|  |  |
| ---------- | ------------- |
| label           | **Abstract** |
| name          |  **abstract** |
| predicate   | **::RDF::Vocab::DC.abstract**

### LINKED ISSUE #84: Add attribute: table_of_contents
|  |  |
| ---------- | ------------- |
| label           | **Table of Contents** |
| name          |  **table_of_contents** |
| predicate   | **::RDF::Vocab::DC.tableOfContents**

### LINKED ISSUE #88: Add attribute: dates - all
|  |  |
| ---------- | ------------- |
| *existing*   | **Date Created** |
| name          | **date_created** |
| predicate   | inherited from Hyrax::BasicMetadata **::RDF::Vocab::DC.date** |
| desc           | *generated text* |
|                    |                              |
| *existing*   | **Date Modified** |
| name          | **date_modified** |
| predicate   | inherited from Hyrax::CoreMetadata **::RDF::Vocab::** |
| desc           | *generated text* |
|                    |                              |
| *existing*   | **Date Submitted** |
| name          | **date_uploaded** |
| predicate   | inherited from Hyrax::CoreMetadata  **::RDF::Vocab::DC.dateSubmitted**|
| desc           | *generated text* |
|                    |                              |
|                    |                              |
| label           | **Primary Date** |
| name          | **date** |
| predicate   | **::RDF::Vocab::DC11.date** |
| desc           | A general date associated with the work if no more specific date metadata is available |
|                    |                              |
| label           | **Date Accepted** |
| name          | **date_accepted** |
| predicate   | ::RDF::Vocab::DC.dateAccepted |
| desc           | The date of acceptance of a resource |
|                    |                              |
| label           | **Date Available** |
| name          | **date_availabe** |
| predicate   | ::RDF::Vocab::DC.available |
| desc           | The date a resource was made or becomes available |
|                    |                                |
| label           | **Date Copyrighted** |
| name          | **date_copyrighted** |
| predicate   | ::RDF::Vocab::DC.dateCopyrighted |
| desc           | The date a work entered copyright |
|                    |                              |
| label           | **Date Issued** |
| name          | **date_issued** |
| predicate   | ::RDF::Vocab::EBUCore.dateIssued |
| desc           | The date a resource was issued |

### PR DESCRIPTION
**Pull request by little9**
Closes #81
Closes #82
Closes #83
Closes #84
Closes #88

### CODE REVIEW

**Comment by little9** [comment]
Closes #90

File: app/presenters/hyrax/generic_work_presenter.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module Hyrax
   class GenericWorkPresenter < Hyrax::WorkShowPresenter
-    delegate :displays_in, to: :solr_document
-    delegate :geographic_name, to: :solr_document
+    include Tufts::SharedTermsService
```
Comment (no-reply) [inline app/presenters/hyrax/generic_work_presenter.rb]: I'm not sure I'm following what these `include`s are for, can you help me out? It doesn't look like `SharedTermsService` actually defines any behavior on the presenter or other classes it is included in?

File: app/services/tufts/shared_terms_service.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module Tufts
```
Comment (no-reply) [inline app/services/tufts/shared_terms_service.rb]: Thinking about my question in the other comment, could this just be:

```ruby
module Tufts
  class Terms
    def self.shared_terms
      [:displays_in, # etc...
      ]
    end
  end
end
```

Then, when needing to use the terms do `Tufts::Terms.shared_terms`?

We could also move the terms array to a frozen constant. Right now we allocate the array from scratch each time the method is called.

```ruby
class Terms
  SHARED_TERMS = [:displays_in, # etc...
                                    ].freeze
  def self.shared_terms
    SHARED_TERMS
  end
end
```

File: app/presenters/hyrax/generic_work_presenter.rb (hunk 1 above)
Comment (no-reply) [inline app/presenters/hyrax/generic_work_presenter.rb]: I'm not sure I follow what these includes are doing? Do we want these classes to all inherit this service, or can they just call it as a collaborator?

Comment (little9) [inline app/presenters/hyrax/generic_work_presenter.rb]: I'll switch these around to just call it

**Comment by little9** [comment]
I made the include changes that you suggested and made the array a constant in Tufts::Terms

**Comment by no-reply** [comment]
Thanks. It looks like it's failing over minor style issues. Could you fixup, else I can in the early hours tomorrow.

**Comment by little9** [comment]
Should be good now.. I forgot to run rubocop before pushing :)

**Pull request merged**